tp-api-gateway-dev.yaml

```diff
@@ -1,6 +1,16 @@
 keycloak:
   issuer-uri: ${KEYCLOAK_URL_AUTH:https://keycloak-tsplatform.techsophy.com}/auth/realms/
   realm.name: ${KEYCLOAK_REALM_NAME:techsophy-platform}
+  client.id:  ${CAMUNDA_IDENTITY_SERVICE:camunda-identity-service}
+  auth-server-url: ${KEYCLOAK_URL_AUTH:https://keycloak-tsplatform.techsophy.com}/auth
+  master:
+      username: ${MASTER_USERNAME:venkataramana}
```

Comment:
no default user should be configured in the realm, does this user has access to all the realms?

Reply:
yes vaibhav removing default user